backend/danswer/document_index/vespa/indexing_utils.py

```diff
@@ -200,6 +202,7 @@ def batch_index_vespa_chunks(
     chunks: list[DocMetadataAwareIndexChunk],
     index_name: str,
     http_client: httpx.Client,
+    multitenant: bool,
```

Comment:
Is there reason we're passing multitenant down instead of using the environment variable directly in the couple of places where it's necessary?

Reply:
It's bad style to inject global side effects into functions. We actually need to refactor this kind of behavior out of the codebase everywhere.

https://android.googlesource.com/platform/external/google-styleguide/+/refs/tags/android-s-beta-2/pyguide.md#2_5-global-variables

Reply:
I understand that- what I meant was more along the lines of using the config where it's necessary for kicking off the initial tasks and letting the `tenant_id` value propagate through the various background tenant tasks (which should be `None` if multitenancy is not enabled), rather than passing an additional boolean to each task. 

But looking at the code again, it looks like it's only passed to Vespa for initialization and used for indexing, so no real complaints here.